src/test/JerryProtocolHandler.test.ts

```diff
@@ -15,6 +15,13 @@ function encodeArray(byte: number, str: string) {
   return array;
 }
 
+function getVersion(configArray) {
+  let version = new Uint8Array(4);
+  setUint32(true , version, 0, SP.JERRY_DEBUGGER_VERSION);
+  configArray.splice(2, 0, ...version);
+  return Uint8Array.from(configArray);
+}
```

Comment:
In this case, this is not a lucky name choice. The getter function should not be set anything. I suggest you to return with the version's bytes only. You can use the `concat()` method to join the returned array with more arrays later.